commands2/commandscheduler.py

```diff
@@ -29,7 +38,7 @@ def __new__(cls) -> CommandScheduler:
         return cls._instance
 
     @staticmethod
-    def getInstance() -> "CommandScheduler":
+    def getInstance() -> CommandScheduler:
```

Comment:
Does this work in the oldest python we support?

Reply:
The tests are running in 3.8 in CI so... yes? Isn't that what `from __future__ import annotations` does?